src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py

```diff
@@ -144,13 +157,13 @@ def get_input_strides(self, index: int) -> list:
 
     def get_input_type(self, index):
         if index < len(self.input_types):
-            return OVAny(pt_to_ov_type_map[str(self.input_types[index])])
+            return self.input_types[index]
         input = self._raw_input(index)
         return self.get_type_for_value(input)
 
     def get_output_debug_name(self, index):
-        if self._output_names is not None and index < len(self._output_names):
-            return self._output_names[index]
+        if self._outputs is not None and index < len(self._outputs) and self._outputs[index][0]:
+            return self._outputs[index][0]
```

Comment:
For non dictionary outputs, will this return an empty string?

Reply:
good catch, yes it will, need to check that it is not empty string

Reply:
`self._outputs[index][0]` check that name is empty or not.